src/components/KonnectorItem.jsx

```diff
@@ -1,31 +1,83 @@
+import styles from '../styles/konnectorItem'
+
 import React, { Component } from 'react'
 import { Link, withRouter } from 'react-router'
 import { translate } from 'cozy-ui/react/I18n'
+import { CONNECTION_STATUS } from '../lib/CollectStore'
 
 class KonnectorItem extends Component {
-  render ({ konnector, router }) {
+  constructor (props, context) {
+    super(props, context)
+    this.store = this.context.store
+
+    const { konnector } = this.props
+
+    this.state = {
+      status: this.store.getConnectionStatus(konnector)
+    }
+
+    this.connectionListener = status => {
```

Comment:
`this.connectionListener` is an arrow function, which from a given `status` update the component's state.